packages/grafana-api-clients/src/clients/rtkq/logsdrilldown/v1alpha1/endpoints.gen.ts

```diff
@@ -0,0 +1,617 @@
+import { api } from './baseAPI';
+export const addTagTypes = ['API Discovery', 'LogsDrilldown'] as const;
+const injectedRtkApi = api
+  .enhanceEndpoints({
+    addTagTypes,
+  })
+  .injectEndpoints({
+    endpoints: (build) => ({
+      getApiResources: build.query<GetApiResourcesApiResponse, GetApiResourcesApiArg>({
+        query: () => ({ url: `/apis/logsdrilldown.grafana.app/v1alpha1/` }),
+        providesTags: ['API Discovery'],
+      }),
+      listLogsDrilldown: build.query<ListLogsDrilldownApiResponse, ListLogsDrilldownApiArg>({
+        query: (queryArg) => ({
+          url: `/logsdrilldowns`,
```

Comment:
Is this URL auto-generated or can we use other paths? Because what are "logs drilldowns" plural.

Reply:
This is auto generated.

Reply:
I _think_ it's using [this value](https://github.com/grafana/grafana/blob/54041155bd42888a09337ba790894f701b1ac186/apps/logsdrilldown/pkg/apis/logsdrilldown_manifest.go#L38) (lowercased) for the list endpoints, so it can be customised on backend.